core/fileutil/dir.go

```diff
@@ -0,0 +1,12 @@
+//+build !windows,!darwin
+
+package fileutil
+
+import (
+	"os"
+	"path/filepath"
+)
+
+func DefaultDir() string {
```

Comment:
I would suggest making one more file though, doc.go with the exported function that just calls the "real" one which is unexported.